src/Miunie.ConsoleApp/InversionOfControl.cs

```diff
@@ -36,6 +36,7 @@ private static void InitializeProvider()
                 .AddSingleton<ConfigurationFileEditor>()
                 .AddSingleton<EntityConvertor>() // TODO (Peter): Depricate
                 .AddSingleton<MiunieUserService>()
+                .AddSingleton<LanguageResources>()
```

Comment:
Hide that behind an `ILanguageResources` interface so that we can swap either whole languages or lang versions.